gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/functions/ElementToCSV.java

```diff
@@ -58,8 +61,11 @@ public void open(Configuration parameters) throws Exception {
    */
   String getPropertyString(E element) {
     return metaData.getPropertyMetaData(element.getLabel()).stream()
-      .map(metaData -> element.getPropertyValue(metaData.getKey()))
-      .map(value -> value == null ? EMPTY_STRING : value.toString())
+      .map(propertyMetaData -> {
```

Comment:
IMO anonymous methods should never exceed one line. Please extract a simple map method.

Reply:
Do you mean something like
`.map(propertyMetaData -> getPropertyStringToken(propertyMetaData, element))`
or something different? Can you give me an example?

Reply:
I guess you are in the wrong line? I refer to the lines 64-67

Reply:
Lines 64-67 are exactly what i am talking about. I don't think an extra method is necessary for that 2 line Lambda.

Reply:
I had someone else review the line as well, it took him roughly 3 minutes to understand this small snippet. so for clarification at least a comment is needed which will stretch the method even further. After a small discussion with @ChrizZz110 about this topic we came to the conclusion that it would be save and easier to read to extract this map method. Please provide an update to your PR.

Reply:
You can define a private method for mapping, e.g. 
```
private Object parseValue(PropertyMetaData propertyMetaData) {
    PropertyValue p = propertyMetaData.getPropertyValue(propertyMetaData.getKey());

    if (p != null && getTypeString(p).equals(propertyMetaData.getTypeString()) {
      return p.toString();
    }
    return EMPTY_STRING;
  }
```
then you can use it in map() function:
`.map(this::parseValue)`

Reply:
See [`MetaDataParser::parseMapProperty`](https://github.com/dbs-leipzig/gradoop/blob/master/gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/metadata/MetaDataParser.java#L251)

Reply:
I need `element` to use `element.getPropertyValue(propertyMetaData.getKey())`. How can i use additional parameters here?

Reply:
In that case you have to add a 2nd parameter and call the method with (propertyMetaData -> this.parseValue(propertyMetaData, element))

Reply:
exactly this solution has already been proposed by timo95 ^^

Reply:
In the first comment? Yes, I think I got you wrong there. So this would fit well.